pkg/acs/calacs/calacs/calacs.c

```diff
@@ -611,7 +612,7 @@ int ProcessACSCCD (AsnInfo *asn, ACSInfo *acshdr, int *save_tmp, int printtime,
                 if (acshdr->sci_basic_cte == PERFORM) {
                     if (ACScte(acshdr->blv_tmp, acshdr->blc_tmp,
                                &acscte_sci_sw, &sciref, printtime,
-                               asn->verbose, nThreads, cteAlgorithmGen, pcteTabNameFromCmd)) { //this is the line
+                               asn->verbose, nThreads, cteAlgorithmGen, pcteTabNameFromCmd, false)) { //this is the line
```

Comment:
I would like to see an explicit comment here indicating the forwardModelOnly is not a feature that was designed to be used during pipeline processing (hence the false).  Rather this option is for standalone processing when running the CTE component of the pipeline for calibration purposes.